Data-Engineering-CYA/README.md

```diff
@@ -0,0 +1,57 @@
+# Choose Your Own Adventure
+Welcome to the adventure into the Data team at Vox Media. Our goal is to become a world-class data engineering team leading the way in developing production-ready and state-of-the-art data science solutions. We do this by providing Vox Media Stakeholders with democratized and integrated data solutions that help them gain clarity, insights, and scale with first-party data. 
+## How To Prepare
+For the Data Engineer role, you will participate in a 45-technical panel interview with the Revenue Data team. Choose any ONE of the following prompts and come prepared to present and discuss. 
+
+Spend a **maximum of 2 hours** on this part of the choose your own adventure exercise. 
+## Technical Prompts 
+### Prompt One: Around the world in 45 minutes?
+This is wild west style and we’re taking a trip around the Data Engineering world with stops at the following attractions:
+
+**The Zoo of Serpents:** SQL and Python
+* What is your level of experience with SQL and Python
+* What tools have you used in the past to develop data solutions
+* Have you worked in ML teams before? What libraries are used to label and clean datasets?
+
+**Le Data Modelé:** for Data Warehousing and Big Data
+* Explain why staging areas are useful
+* What is Data Mining
+
+**Express Train Lane:** Or just… ETL
+* Structured versus unstructured data
+* What is ETL?
+* What happens after a data pipeline is available?
+
+**Institute of Mathematica:**
+* * What the benefits of using MapReduce functions
+* * Explain the Pub/Sub model 
+
+**The Museum of Visual Arts:**
+* * Software engineering background or experience
+* * Do you have experience or an interest in Javascript or Data visualization?
```

Comment:
In all of the above, can we add question marks on all the questions? And turn the phrases into complete sentences. Making it shiny ✨